bowling/bowlingLane.cpp

```diff
@@ -1,33 +1,53 @@
 #include "bowlingLane.hpp"
 
+#include <algorithm>
 #include <iostream>
 #include <memory>
 
-BowlingLane::BowlingLane() {}
+// BowlingLane::BowlingLane() {}
 
-BowlingLane::BowlingLane(File file) {
-    file = File(name_);
-}
+BowlingLane::BowlingLane(std::vector<std::unique_ptr<Player>> players)

```

Comment:
take const & to vector to avoid additional copy or && (you already have std::move). Without && std::move moves a copy, so there is no benefit of having it.